tests/integration/test_interaction_assessment.py

```diff
@@ -150,6 +160,86 @@ def test_max_attempts_reached_submit_and_reset_disabled(self):
         self.assertEqual(submit_button.get_attribute('disabled'), 'true')
         self.assertEqual(reset_button.get_attribute('disabled'), 'true')
 
+    def test_show_answer(self):
+        """
+        Test "Show Answer" button is shown in assessment mode, enabled when no
+        more attempts remaining, and displays correct answers when clicked.
+        """
+        show_answer_button = self._get_show_answer_button()
+        self.assertTrue(show_answer_button.is_displayed())
+
+        self.place_item(0, TOP_ZONE_ID)
+        for _ in xrange(self.MAX_ATTEMPTS):
+            self.assertEqual(show_answer_button.get_attribute('disabled'), 'true')
+            self.click_submit()
+
+        self.assertIsNone(show_answer_button.get_attribute('disabled'))
+        self.click_show_answer()
+
+        zones = dict(self.all_zones)
+        for item in self._get_items_with_zone(self.items_map).values():
+            zone_titles = [zones[zone_id] for zone_id in item.zone_ids]
+            # When showing answers, correct items are placed as if assessment_mode=False
+            self.assert_placed_item(item.item_id, zone_titles, assessment_mode=False)
```

Comment:
@arbrandes This block is duplicated below. Factor it out into a separate `_assert_show_answer_item_placement` method, perhaps? When you do, keep the inline comment from L182 please, it's very useful :)

It might make sense to include the `for` loop below as well. That way, both tests would be checking the placement of decoy items.